JourneysSquad/JourneysSquad/HistoryOfOperations/BillListView.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftUI
+
+/// A view that displays a list of bills grouped by date.
+struct BillListView: View {
+    var bills: BillsModel = .init()
```

Comment:
```suggestion
    var bills =  BillsModel ()
```
I suggest to use simpler syntax.